permissions.json

```diff
@@ -0,0 +1,33 @@
+{ 
```

Comment:
I think enums are more suited for permissions

Reply:
Whoops. This was a file I used to lay them out in my head. It doesn't actually do anything

Reply:
Deleted it